src/Components/SalesforceWrapper.tsx

```diff
@@ -1,21 +1,27 @@
 import { getENV } from "Utils/getENV"
 import { useLoadScript } from "Utils/Hooks/useLoadScript"
 import { useAppendStylesheet } from "Utils/Hooks/useAppendStylesheet"
-import { themeGet } from "@styled-system/theme-get"
 
 export const SalesforceWrapper: React.FC = () => {
   useAppendStylesheet({
     id: "salesforce-chat-styles",
     body: `
-    .embeddedServiceHelpButton .helpButton .uiButton {
-      background: ${themeGet("colors.black100")};
-      font-family: ${themeGet("fonts.sans")};
-      box-shadow: none;
-    }
-  
-    .embeddedServiceHelpButton .helpButton .uiButton:focus {
-      background: ${themeGet("colors.blue100")};
-    }
+      .embeddedServiceHelpButton .helpButton .uiButton {
+        background: #000000; /* colors.black100 */
+        font-family: "ll-unica77", "Helvetica Neue", Helvetica, Arial, sans-serif; /* fonts.sans */
+        box-shadow: none;
+      }
+    
+      .embeddedServiceHelpButton .helpButton .uiButton:focus {
+        background: #1023D7; /* colors.blue100 */
+      }
+      
+      /* Make sure it doesn't render when loaded on desktop and then the window is resized */
+      @media (max-width: 767px) { /* mediaQueries.xs */
```

Comment:
is it really necessary? Doesn't hurt having it but we don't need to cover a lot of edge cases from resizing Deskop<->Mobile, usually the ones doing it are us, not the final users.

Reply:
I'd say better safe than sorry on this one! Given the screenshots that partners share with us, very often they do not work with the window full size (and the same goes for some Artsy members as well, even non-engineers)